From: David Paleino <d.paleino@gmail.com>
Subject: fix deprecated include
Forwarded: no

---
 include/mapnik/feature.hpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- mapnik.orig/include/mapnik/feature.hpp
+++ mapnik/include/mapnik/feature.hpp
@@ -29,7 +29,7 @@
 #include <mapnik/geometry.hpp>
 #include <mapnik/raster.hpp>
 // boost
-#include <boost/property_map.hpp>
+#include <boost/property_map/property_map.hpp>
 #include <boost/utility.hpp>
 #include <boost/shared_ptr.hpp>
 // stl
