From: David Paleino <dapal@debian.org>
Subject: fix variable name when the architecture is big endian
Origin: vendor
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=644704
Forwarded: no

---
 include/mapnik/grid/grid.hpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- mapnik.orig/include/mapnik/grid/grid.hpp
+++ mapnik/include/mapnik/grid/grid.hpp
@@ -284,7 +284,7 @@ public:
         {
 
 #ifdef MAPNIK_BIG_ENDIAN
-            unsigned a1 = (int)((rgba1 & 0xff) * opacity) & 0xff; // adjust for desired opacity
+            unsigned a = (int)((rgba1 & 0xff) * opacity) & 0xff; // adjust for desired opacity
 #else
             unsigned a = (int)(((rgba1 >> 24) & 0xff) * opacity) & 0xff; // adjust for desired opacity
 #endif                    
